### PR TITLE
Adds Ice Barrage

### PR DESCRIPTION
**Pull request by MadmanMartian**
:cl:
 * rscadd: Adds the Ice Barrage spell. Freezes the target in a block of ice for 10 seconds

### CODE REVIEW

**Comment by yclatious** [comment]
Absolute madman.

**Comment by boy2mantwicethefam** [comment]
Absolute madman.

File: code/game/objects/structures/ice_block.dm [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		spawn(lifetime)
+			melt()
+
+/obj/structure/ice_block/fire_act(datum/gas_mixture/air, exposed_temperature, exposed_volume)
+	if(exposed_temperature > T0C + 100)
+		melt()
+
+/obj/structure/ice_block/examine(mob/user)
+	..()
+	var/atom/A = get_locked(/datum/locking_category/ice_block)[1]
```
Comment (DamianX) [inline code/game/objects/structures/ice_block.dm]: out of range runtime?

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Needs changelog and description

**Comment by SonixApache** [comment]
Absolute madman.

**Comment by small-boss** [comment]
protection prayers when

**Comment by Chazach** [comment]
Does this just paralyse the guy or is it like stone touch where you can smash the statute and the guy inside will die?

**Comment by MadmanMartian** [comment]
It stuns you, and then you're locked to the ice block

If you can destroy the ice block, you can get them out.

**Comment by MadmanMartian** [comment]
freaking hate binary conflicts

**Pull request merged**